### PR TITLE
New dev build

### PR DESCRIPTION
**Pull request by cuahsimarko**
### Pull Request Checklist: 
- [x] Positive Test Case Written by Dev

- [x] Automated Testing

- [x] Sufficient User and Developer Documentation

- [x] Passing Jenkins Build

- [x] Peer Code review and approval

This PR is to update files that impact early adopters only. Folks, who do not use alternate-compose.yml, will not be affected by changes in this PR. It updates a number of aspects for early adopters of the enhanced docker workflow at https://cuahsi.atlassian.net/wiki/x/A4BS

### CODE REVIEW

File: Dockerfile-defaultworker [hunk 1]
```diff
@@ -4,6 +4,7 @@ MAINTAINER Phuong Doan pdoan@cuahsi.org
 
 # inplaceedit in pip doesn't seem compatible with Django 1.11 yet...
 RUN pip install git+https://github.com/theromis/django-inplaceedit.git@e6fa12355defedf769a5f06edc8fc079a6e982ec
+RUN pip install markdown
```
Comment (sblack-usu) [inline Dockerfile-defaultworker]: remove this and update hs_docker_base to release-1.9.3

File: hydroshare/local_settings.template [hunk 2]
```diff
@@ -61,15 +171,37 @@ HS_IRODS_LOCAL_ZONE_DEF_RES='hydroshareLocalResc'
...
+MAILCHIMP_SUBSCRIBERS = "f0c27254e3"
+
+# sendfile support for large files
+# These must match settings in nginx
 SENDFILE_ON = False
 IRODS_USER_URI = "/irods-user"
 IRODS_DATA_URI = "/irods-data"
 LOCAL_CACHE_URI = "/local-cache"
 
-# Needed for deployments to non-dev environments
```
Comment (sblack-usu) [inline hydroshare/local_settings.template]: This wasn't the best comment, but I think it's worth putting in a comment here that says these keys are test keys.

File: Dockerfile-defaultworker (hunk 1 above)
Comment (cuahsimarko) [reply Dockerfile-defaultworker]: Fixed in 3e85239f2

File: hydroshare/local_settings.template [hunk 3]
```diff
@@ -61,15 +171,37 @@ HS_IRODS_LOCAL_ZONE_DEF_RES='hydroshareLocalResc'
...
+RECAPTCHA_SITE_KEY="6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"
+RECAPTCHA_SECRET_KEY="6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"
+RECAPTCHA_VERIFY_URL='https://www.google.com/recaptcha/api/siteverify'
+
 with open(os.path.dirname(os.path.abspath(__file__))  + "/../config/hydroshare-config.yaml", 'r') as stream:
```
Comment (sblack-usu) [inline hydroshare/local_settings.template]: You should be able to remove this block.  It's not used yet and was only added to expose some configuration for @phuongdm.  It's not needed in a local dev environment.

File: hydroshare/local_settings.template (hunk 2 above)
Comment (cuahsimarko) [reply hydroshare/local_settings.template]: Fixed in 1e92768ca

File: hydroshare/local_settings.template (hunk 3 above)
Comment (cuahsimarko) [reply hydroshare/local_settings.template]: Fixed in 1e92768ca

**Comment by cuahsimarko** [comment]
retest this, yo

**Comment by cuahsimarko** [comment]
retest this, please

**Comment by cuahsimarko** [comment]
thank you, jenkins, you the best

**Comment by sblack-usu** [comment]
so please is necessary?

**Comment by cuahsimarko** [comment]
Yes, please is actually necessary. I chuckle at the thought that the author may have put into that.

**Pull request merged**